src/main/java/core/AbstractParameters.java

```diff
@@ -25,12 +25,14 @@ public abstract class AbstractParameters {
     long incrementMilestoneS = 0;
 
 
-    public AbstractParameters(long seed) {
```

Comment:
A question on this - if we do want to run with a specific random seed, should a second constructor specifying this not be included?

Reply:
There is a public method setRandomSeed(long seed) which can be used in this use case.